src/photos/components/ErrorUnshared.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ErrorShare from 'components/Error/ErrorShare'
+import { Main } from 'cozy-ui/react/Layout'
+import styles from '../targets/public/index.styl'
+import classNames from 'classnames'
+
+const ErrorUnsharedComponent = () => (
+  <div
+    className={classNames(
+      styles['pho-public-layout'],
+      styles['pho-public-layout--full'],
+      'u-pt-3'
+    )}
+  >
+    <Main className="u-pt-1-half">
+      <ErrorShare errorType={`public_album_unshared`} />
```

Comment:
why not `errorType="public_album_unshared"` ?

Reply:
As already said, I really like the `={}` syntax because I find it more consistent with the rest of the code : 
boolean={true/false}
object={object}
string={'string'} 

;)